### DISCUSSION TITLE
Freeplane Docs site change

### DISCUSSION THREAD

**Discussion by dpolivaev** [freeplane/freeplane#645] [Announcements]
Hello,

it turned out that our recent version of the documentation site could not be indexed by Google.
I had to replace the previous renderer with a static page generator.

Please have a look at the new version and comment.

https://docs.freeplane.org/

Cheers,
Dimitry

**Comment by euu2021** [comment]
I already noticed that the search is better than in the previous site.

Let's monitor the google indexing. Nothing, yet:

![image](https://user-images.githubusercontent.com/77707706/189461185-dc8a0f16-b16d-4c03-9c7f-d040c50d5913.png)

**Reply by euu2021** [reply]
and... the indexing happened 🎉

![image](https://user-images.githubusercontent.com/77707706/212983055-e46dda7c-6fcf-4412-87e6-98e21bffb091.png)

**Comment by rubisara38** [comment]
Hi @dpolivaev ,

The [Scripting API page](https://www.freeplane.org/doc/api/org/freeplane/plugin/script/proxy/Proxy.NodeRO.html) has been down for some time.

Could you please take a look? Thank you.

**Reply by dpolivaev** [reply]
I intentionally removed it because it is not up-to-date. The API is distributed with Freeplane itself and it is available in the help menu.

**Reply by EdoFro** [reply]
Hi,
I think it is good to also have an up-to-date Scripting API Page:
Sometimes, when answering scripting questions in the forum, it is useful to have a way to add a link to the api documentation.

it's not urgent, but may be useful in the future.

**Reply by EdoFro** [reply]
FYI,
@macmarrum has [added the API to the documentation page](https://github.com/freeplane/docs/issues/102#issuecomment-1487039982)

https://docs.freeplane.org/api/
